packages/eslint-plugin-ecmascript-compat/lib/delegation.js

```diff
@@ -1,5 +1,6 @@
 function createDelegatee(config, rootContext) {
   const { definition, options } = config;
+  const pollyfils = (rootContext.options && rootContext.options[0] && rootContext.options[0].polyfills) || [];
```

Comment:
Note from `package.json:engines` (the non-root ones) that Node 14 is required, so optional chaining and nullish coalescing operators are available